src/cli/MeasureUpdateCommand.cpp

```diff
@@ -88,6 +97,203 @@ namespace cli {
+      std::vector<std::string> firstLevelTaxonomyTerms = openstudio::BCLMeasure::suggestedFirstLevelTaxonomyTerms();
+      taxonomyTags.reserve(35);  // 2023-05-03: at this moment, this is the total number
+      for (auto& firstLevelTaxonomyTerm : firstLevelTaxonomyTerms) {
+        auto secondLevelTaxonomyTerms = openstudio::BCLMeasure::suggestedSecondLevelTaxonomyTerms(firstLevelTaxonomyTerm);
+        taxonomyTags.reserve(taxonomyTags.size() + secondLevelTaxonomyTerms.size());
+        std::transform(secondLevelTaxonomyTerms.begin(), secondLevelTaxonomyTerms.end(), std::back_inserter(taxonomyTags),
+                       [&firstLevelTaxonomyTerm](auto&& secondLevelTaxonomyTerm) {
+                         return fmt::format("{}.{}", firstLevelTaxonomyTerm, secondLevelTaxonomyTerm);
+                       });
+      }
+
+      newMeasureSubCommand
+        ->add_option("--taxonomy-tag", opt->newMeasureOpts.taxonomyTag, fmt::format("Taxonomy Tag [Default: '{}']", opt->newMeasureOpts.taxonomyTag))
+        ->option_text("tag")
+        ->required(false)
+        ->check(CLI::IsMember(taxonomyTags))
+        ->capture_default_str();
+      newMeasureSubCommand->add_option("-d,--description", opt->newMeasureOpts.description, "Description")->option_text("desc")->required(false);
+      newMeasureSubCommand->add_option("-m,--modeler-description", opt->newMeasureOpts.modelerDescription, "Modeler Description")
+        ->option_text("modeler_desc")
+        ->required(false);
+
+      [[maybe_unused]] auto* directoryPathOpt =
+        newMeasureSubCommand->add_option("DIRECTORY", opt->newMeasureOpts.directoryPath, "Directory for the measure")
+          ->option_text(" ")
+          ->required(true)
+          ->check(CLI::NonexistentPath);
+
+      {
+        static constexpr auto helpOptionsGroupName = "Help";
+        static constexpr auto newMeasureExamples = R"(Examples:
+
+1. Create a Ruby ModelMeasure:
+
+```
+$ openstudio labs measure new --class-name MyExampleRubyModelMeasure
+$ openstudio labs measure new --class-name MyExampleRubyModelMeasure --type ModelMeasure --language Ruby
+```
+
+2. Pass all optional args to create a Python EnergyPlusMeasure:
+
+```
+$ openstudio labs measure new --class-name MyExamplePythonMeasure --type EnergyPlusMeasure --language Python --name "My Python Measure" --description "This is my measure" --modeler-description "This does complicated stuff" --taxonomy-tag "Envelope.Form" ./test_measure
+
+$ openstudio labs measure new -c MyExamplePythonMeasure -t EnergyPlusMeasure -l Python -n "My Python Measure" -d "This is my measure" -m "This does complicated stuff" --taxonomy-tag "Envelope.Form" ./test_measure
+```
+
+3. List taxonomy tags
+```
+$ openstudio labs measure new --list-taxonomy-tags
+$ openstudio labs measure new --list-for-first-taxonomy-tag HVAC
+```
+)";
+        newMeasureSubCommand->set_help_flag("-h,--help", "Print this help message and exit")->group(helpOptionsGroupName);
```

Comment:
Since I'm making a specific "Help" group, might as well make the CLI11 default '--help' fall into that group.